server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/AutoDbMigration.java

```diff
@@ -75,6 +77,27 @@ void installH2() {
   @VisibleForTesting
   protected void createSchema(Connection connection, String dialectId) {
     DdlUtils.createSchema(connection, dialectId, false);
+    hackFixForProjectMeasureTreeQueries(connection);
+  }
+
+  private void hackFixForProjectMeasureTreeQueries(Connection connection) {
+    int metricId = 1;
+    try (PreparedStatement preparedStatement = connection.prepareStatement("insert into PROJECT_MEASURES (METRIC_ID,COMPONENT_UUID,ANALYSIS_UUID) values (?,?,?);")) {
+      for (int i = 1; i < 1000; i++) {
+        preparedStatement.setInt(1, metricId);
+        preparedStatement.setString(2, "foo_" + i);
+        preparedStatement.setString(3, "bar_" + i);
+        preparedStatement.addBatch();
+        if (i % 250 == 0) {
+          preparedStatement.executeBatch();
+          connection.commit();
+        }
+      }
+      preparedStatement.executeBatch();
+      connection.commit();
+    } catch (SQLException e) {
+      throw new RuntimeException("Failed to insert fake rows into table PROJECTS", e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00112)